kitchen-sink-demo/app/1.chunk.js

```diff
@@ -1,2 +0,0 @@
-webpackJsonp([1],{399:function(e,t,n){"use strict";var o=this&&this.__decorate||function(e,t,n,o){var a,r=arguments.length,c=r<3?t:null===o?o=Object.getOwnPropertyDescriptor(t,n):o;if("object"==typeof Reflect&&"function"==typeof Reflect.decorate)c=Reflect.decorate(e,t,n,o);else for(var f=e.length-1;f>=0;f--)(a=e[f])&&(c=(r<3?a(c):r>3?a(t,n,c):a(t,n))||c);return r>3&&c&&Object.defineProperty(t,n,c),c},a=this&&this.__metadata||function(e,t){if("object"==typeof Reflect&&"function"==typeof Reflect.metadata)return Reflect.metadata(e,t)},r=n(2),c=n(2),f=n(34),i=function(){function e(){}return e}();i=o([r.Component({template:"<H1>Lazy Loaded Component One</H1>"}),a("design:paramtypes",[])],i);var p=[{path:"lazy1",component:i}],u=function(){function e(){}return e}();u=o([c.NgModule({imports:[f.RouterModule.forChild(p)],declarations:[i]}),a("design:paramtypes",[])],u),t.LazyLoadedModule=u}});
```

Comment:
@santiago-elustondo Can you remove the `chunks.js` and `bundle.js` files from app folder.

Reply:
add to .gitignore ?